pkgs/by-name/sw/sway-scratch/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "sway-scratch";
+  version = "0.2.1";
+  src = fetchFromGitHub {
+    owner = "aokellermann";
+    repo = "sway-scratch";
+    rev = "87df238d13dc15ba5733962c55e363766888724e";
```

```suggestion
    rev = "refs/tags/v${version}";
```

please make sure to recompute the hash as well

---

fixed